test/end-to-end/specs/createBlueprint.test.js

```diff
@@ -44,6 +44,15 @@ describe("Create Blueprints Page", function() {
     expect(createBlueprintPage.createButton.getAttribute("disabled")).to.equal("true");
   });
 
+  it("Invalid character for blueprint name should not be allowed", function() {
+    const invalidName = "`~!@#$%^&*()=+[]{}\\|;:'\",<>/?";
+    createBlueprintPage.nameBox.setValue(invalidName);
```

Comment:
IMHO this is too strong a test case. This should already fail with a name that contains a single invalid character -- it's almost guaranteed to fail on a name that's nothing but garbage :grin: 

I'd test a name which has some funny, but still valid characters (which should be accepted), and then the edge case from the other side by adding one invalid character; and maybe have a few cases of that with different invalid ones.

I realize this is a bit of bikeshedding at this point. But this is a good opportunity for @jkozol to get familiar with the tests, run it locally, and adjust it as above?